docker-compose.yml

```diff
@@ -103,6 +103,18 @@ services:
         '1',
         '--reload',
       ]
+  scheduler:
```

Comment:
not required

Reply:
https://github.com/WorkflowAI/WorkflowAI/pull/161/commits/8899ab56c8c48bba2d6d8effe99cfb866e165752

Reply:
looks like the change is still here ?

Reply:
https://github.com/WorkflowAI/WorkflowAI/pull/161#discussion_r2054040899